package.json

```diff
@@ -14,6 +14,7 @@
     "event-stream": "^3.1.7",
     "express": "^4.9.7",
     "frozen-moment": "^0.4.0",
+    "kerberos": "~0.0",
```

Comment:
Don't use the `~` syntax for packages in `0.0.z` version number: this is akin to having no version contraint at all and puts us at risk of breaking changes. In this case, you need to specify the full version number.